test/cpp/end2end/grpclb_end2end_test.cc

```diff
@@ -1899,6 +1886,19 @@ TEST_F(SingleBalancerTest, DropAll) {
   EXPECT_EQ(status.error_message(), "drop directed by grpclb balancer");
 }
 
+TEST_F(SingleBalancerTest, SecureNamingDeathTest) {
```

Comment:
Can you add a comment letting us know to keep it at the end?

Reply:
Done